src/components/manage/Tiles/Maps/View.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * View map tile.
+ * @module components/manage/Tiles/Maps/View
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image, Container } from 'semantic-ui-react';
+
+/**
+ * View image tile class.
+ * @class View
+ * @extends Component
+ */
+const View = ({ data }) => (
+  <p
+    className={['tile', 'image', 'align', data.align]
+      .filter(e => !!e)
+      .join(' ')}
+  >
+    <Container >
+    <iframe src={data.url} width="600" height="450" frameborder="0" style={{border:0, zIndex: 1, position: 'relative' }} allowfullscreen></iframe>
```

Comment:
How about responsive views? we have to place the width and height (and maybe the style attrs as well) as a CSS class so we can style properly the responsive viewports.